samza-core/src/main/scala/org/apache/samza/container/TaskInstance.scala

```diff
@@ -274,7 +274,11 @@ class TaskInstance(
 
     if (storageManager != null) {
       trace("Remove old checkpoint stores for taskName: %s" format taskName)
-      storageManager.removeOldCheckpoints(checkpointId)
+      try {
+        storageManager.removeOldCheckpoints(checkpointId)
+      } catch {
+        case e: Exception => error("Failed to remove old checkpoints", e)
```

Comment:
Minor: include current checkpoint id and task name in message.

Reply:
ack, i'll add those